enterprise/api/v1/views/enterprise_group.py

```diff
@@ -118,9 +119,15 @@ def get_learners(self, *args, **kwargs):
         """
 
         group_uuid = kwargs.get('group_uuid')
+        # so our endpoint would look something like this
+        # GET api/v1/enterprise-group/<group_uuid>/learners/?filter=pending_learner_id
+        filter_for_pending_memberships = kwargs.get('pending_learner_id')
+
         try:
             group_object = self.get_queryset().get(uuid=group_uuid)
             members = group_object.get_all_learners()
+            if filter_for_pending_memberships is not None:
+                members = members.filter(learner_uuid=None)
```

Comment:
For performance reasons, I would probably pass a bool into `get_all_learners` and update the DB query